modules/150-user-authn/images/dex-authenticator/werf.inc.yaml

```diff
@@ -0,0 +1,95 @@
+---
+image: {{ .ModuleName }}/{{ .ImageName }}
+fromImage: common/distroless
+import:
+  - image: {{ $.ModuleName }}/{{ $.ImageName }}-oauth2-proxy-artifact
+    add: /src/oauth2-proxy
+    to: /usr/local/bin/oauth2_proxy
+    before: setup
+  - image: {{ $.ModuleName }}/{{ $.ImageName }}-url-exec-prober-artifact
+    add: /src/url-exec-prober
+    to: /usr/local/bin/url-exec-prober
+    before: setup
+docker:
+  ENTRYPOINT: [ "/bin/oauth2_proxy" ]
```

Comment:
you copy oauth2-proxy to /usr/local/bin but trying to start from /bin

Reply:
fixed